### PR TITLE
feat: add validation for SignUp and Login modal

### PR DESCRIPTION
**Pull request by VladPlakhotnik**
It remains only to add a POEditor

### CODE REVIEW

**Comment by neverwanalive** [comment]
мда ну и хуита

**Comment by VladPlakhotnik** [comment]
?

File: src/components/SignUp/SignUp.component.tsx [hunk 1]
```diff
@@ -1,55 +1,43 @@
...
-        value={formik.values.repeatPassword}
-      />
-    </Form>
+    <Formik
+      initialValues={initialValues}
+      validationSchema={validationSchema}
+      onSubmit={() => {}}
+    >
+      <Form>
+        <FormContainer flexDirection='column'>
```
Comment (ordinaryask) [inline src/components/SignUp/SignUp.component.tsx]: u can use simple Flexbox

File: src/components/Login/Login.component.tsx [hunk 2]
```diff
@@ -1,38 +1,36 @@
...
-          value={formik.values.username}
-        />
-        <label htmlFor='password'>{t('common:password')}</label>
-        <input
-          id='password'
-          name='password'
-          onChange={formik.handleChange}
-          value={formik.values.password}
-        />
+        <FormContainer flexDirection='column'>
```
Comment (ordinaryask) [inline src/components/Login/Login.component.tsx]: u can use simple Flexbox

**Review by ordinaryask — CHANGES_REQUESTED** [review]

**Review by ordinaryask — APPROVED** [review]

**Pull request merged**